presto-docs/src/main/sphinx/security/hive.rst

```diff
@@ -0,0 +1,309 @@
+===========================
```

Comment:
Move this file to `/connector/hive-security.rst` and change the title to "Hive Connector Security", similar to the "Kafka Tutorial". That will put it next to Hive connector documentation.